.nuke/parameters.ci.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "./build.schema.json",
+  "Solution": "Remotion.sln",
+  "Target": [ "CIBuild" ],
+  "TestConfigurations": [ "Debug" ],
+  "TestExecutionRuntimes": [ "Docker_Win_NET48", "EnforcedLocalMachine" ],
+  "TestPlatforms": [ "x86" ],
+  "TestTargetRuntimes": [ "NET48" ],
+  "TestBrowsers": [ "NoBrowser", "Chrome" ],
+  "TestSqlServers": [ "NoDB", "SqlServer2016" ],
+  "TestFilter": "TestCategory!=LongRunning"
```

Comment:
do yoy have the nunit ticket to watch for me?

Reply:
https://github.com/nunit/nunit/issues/4589